node/test/lib/relay_network.js

```diff
@@ -199,9 +204,11 @@ RelayNetwork.prototype.setCluster = function setCluster(cluster) {
                         cn: serviceName
                     },
                     hasNoParent: true
-                },
-                peers: self.topology[serviceName] // TODO this really ought to be linked directly to the root peers
+                }
             });
+            // The subchannel's peers are linked to the peer list as managed by
+            // incoming connections from the relays.
+            subChannel.peers = channel.peers;
```

Comment:
Hmm okay so after sleuthing with Kris, I agree that what autobahn-node-client is currently doing is simply wrong... but I'm not sure if this way of doing it is right...